### PR TITLE
Added course in nav bar

### LINKED ISSUE #59: Link the Course in NavBar to YouTube Directory of the courses [bug]
### Describe the bug

It would be better to add the Drop-Down for all the courses with respective Hyper-Link to the directory where resources are present

### To Reproduce

1. go to navbar.js file in src/component/navbar directory 
2. add the courses with hyperlink

### Expected Behavior

shows the Hyperlink with respective course Drop down menu in course tab of navbar.

### Screenshot/ Video

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Anuj-Kumar-AJ**
## Fixes Issue

closes #59 

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/components/navbar/Navbar.js [hunk 1]
```diff
@@ -14,20 +17,36 @@ const Navbar = () => {
...
         <li className='navbar-item'><a href="/">About</a></li>
+        <li className='navbar-item'>
+            <Dropdown>
+              <Dropdown.Toggle variant="success" id="dropdown-basic">
+                <h1>Courses</h1>
+              </Dropdown.Toggle>
+              
+
+              <Dropdown.Menu>
+                <Dropdown.Item href="https://www.youtube.com/watch?v=rZ41y93P2Qo&list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ">DSA + JAVA + Interview Preparation</Dropdown.Item>
```
Comment (shivam-sharma7) [inline src/components/navbar/Navbar.js:34]:
```suggestion
                <Dropdown.Item href="https://www.youtube.com/watch?v=rZ41y93P2Qo&list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ">Java + DSA</Dropdown.Item> 
```

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]
I think, Java + DSA would be better. review/ @kaiwalyakoparkar

**Comment by AnuragThePathak** [comment]
You need to update the fill the slots in the template properly.

**Comment by kaiwalyakoparkar** [comment]
@Anuj-Kumar-AJ please complete the pr tempalte as suggested by Anurag above

**Comment by Anuj-Kumar-AJ** [comment]
hi @kaiwalyakoparkar, i am sorry but i don't understand about the PR template suggestion is it the checks I have to mark on the change proposed section? If that's the case I have done that right now. Also please tell I have just used the suggestion of my previous code review in this PR only. does that mean the changes I have performed in this PR is the same as if I have created a new PR with all the changes?  I am new to GitHub please do tell me about it :)

**Comment by AnuragThePathak** [comment]
@Anuj-Kumar-AJ kindly mention which issue your PR fixes.

**Comment by Anuj-Kumar-AJ** [comment]
@AnuragThePathak , my PR resolves issue #59

**Comment by AnuragThePathak** [comment]
@Anuj-Kumar-AJ for your ease, I have updated the description of PR to add the issue number. But the next time, make sure you do not it properly while creating a PR against an issue. It is applicable to any project.

Secondly, kindly provide screenshots of before and after so reviewers can easily figure out what changes your PR brings.

**Comment by Anuj-Kumar-AJ** [comment]
Thanks for the feedback @AnuragThePathak. I will keep that in mind :)

**Comment by AnuragThePathak** [comment]
You need to resolve conflicts to get it merged.

**Comment by Anuj-Kumar-AJ** [comment]
this feature done not support responsive rendering for small screen sites, like mobile or tab

**Pull request closed without merging**